.github/workflows/CI.yml

```diff
@@ -493,50 +451,68 @@ jobs:
             VERSION
           key: e2etest-release-${{ github.run_id }}
 
+      - name: restore cached rpms
+        uses: actions/cache@v2
```

Comment:
Should there be a cache miss check here?

Reply:
IMO its not a good idea with the same reasons as cache for SSM package.